test/commands/distribute/releases/edit-notes-test.ts

```diff
@@ -0,0 +1,202 @@
+
+      testCommandSuccess(result, executionScope);
+    });
+
+    it("if upload edit notes from file", async () => {
+      // Arrange
+      const executionScope = _.flow(setupReleaseDetailsResponse, setupUpdateReleaseResponse)(Nock(fakeHost));
+      const releaseFilePath = createFile(tmpFolderPath, releaseNotesFileName, releaseNotes);
+
+      // Act
+      const command = new EditNotesReleaseCommand(getCommandArgs([releaseIdOption, fakeReleaseId, "--release-notes-file", releaseFilePath]));
+      const result = await command.execute();
+
+      testCommandSuccess(result, executionScope);
+    });
+  });
+
+  context("failed", () => {
+    it("if --release-id option wasn't specified", async () => {
+      // Arrange
+      let errorMessage: string;
+      const expectedErrorMessage = "Missing required option -r / --release-id";
+
+      let command;
+      // Act
+      try {
+        command = new EditNotesReleaseCommand(getCommandArgs([releaseIdOption]));
+      } catch (e) {
+        errorMessage = e.message;
+      }
+
+      // Assert
+      expect(command).to.eql(undefined);
+      expect(errorMessage).to.eql(expectedErrorMessage);
+    });
+
+    it("if --release-id option value is not valid", async () => {
+      // Arrange
+      const expectedErrorMessage = "notanumber is not a valid release id";
+
+      // Act
+      const command = new EditNotesReleaseCommand(getCommandArgs([releaseIdOption, "notanumber"]));
+      const result = await command.execute();
+
+      // Assert
+      expect((result as CommandFailedResult).errorMessage).to.eql(expectedErrorMessage);
+    });
+
+
+    it("if failed to load release details", async () => {
+      // Arrange
+      const expectedErrorMessage = "failed to load release details";
+      const executionScope = setupReleaseDetailsServiceUnavailableResponse(Nock(fakeHost));
+
+      // Act
+      const command = new EditNotesReleaseCommand(getCommandArgs([releaseIdOption, fakeReleaseId]));
+      const result = await command.execute();
+
+      // Assert
+      expect((result as CommandFailedResult).errorMessage).to.eql(expectedErrorMessage);
+      testCommandFailure(executionScope);
+    });
+
+    it("if --release-notes and --release-notes-file both specified", async () => {
+      // Arrange
+      let errorMessage: string;
+      const expectedErrorMessage = "'--release-notes' and '--release-notes-file' switches are mutually exclusive";
+      const releaseFilePath = createFile(tmpFolderPath, releaseNotesFileName, releaseNotes);
+
+      let command;
+      // Act
+      try {
+        command = new EditNotesReleaseCommand(getCommandArgs([releaseIdOption, fakeReleaseId, "--release-notes", releaseNotes, "--release-notes-file", releaseFilePath]));
+      } catch (e) {
+        errorMessage = e.message;
+      }
+
+      // Assert
+      expect(command).to.eql(undefined);
+      expect(errorMessage).to.eql(expectedErrorMessage);
+    });  
+    
```

Comment:
trailing spaces here too

Reply:
Btw it's a lint check?

Reply:
Oh no, just an observation when testing the code locally.